scss/utilities/mixins/_text-decoration.scss

```diff
@@ -73,7 +73,10 @@
 
   &::after {
     @extend %dash-under;
-    @include center-transform(y);
+    // @include center-transform(y);
+    top: .7em;
+    margin-top: auto;
+    margin-bottom: auto;
     background-color: color($dash-color);
```

Comment:
I am guessing that center-transform(y) mixin centers this item vertically? 

If this is the case we technically shouldn't need margin-top and margin-bottom set to auto.  If both are needed and I am misunderstanding the center-transform(y) mixin reference, then no worries!

See the reference:
@mixin center-transform ($axis: false) { // or x or y
  position: absolute;

  @if $axis != x {
    top: 50%;
    margin-top: auto;
    margin-bottom: auto;
  }

  @if $axis != y {
    left: 50%;
    margin-left: auto;
    margin-right: auto;
  }

Reply:
Let's re-look at this later for sure